acquia/cli
# DG-25844. DON'T MERGE YET!!! :)

I'm just pre-creating this PR ahead of time so I don't forget to do it.   See details below

**Motivation**
Currently (from #906 ), the composer script in acquia/cli is pinned to the last known-working branch of gardener repo (the most recent release where the spec generator was known to be working).   The intent of having this composer script exist is that it can run at any time, and will automatically pick up the latest good version from ACSF (gardnerer) master and regen the spec based on whatever is present in API code.  

**Proposed changes**
This PR changes the pinned release version back to master.  BEWARE: the generator is currently still broken in master, but is expected to be fixed by DG-25880, which is expected to be in 2.143 after ACSF 2022 sprint 9.  If we merge and release this change right now, the spec regeneration will not work until gardener fixes are released (but that might not be problematic)

**Alternatives considered**


**Testing steps**
Run the following:
```
composer  update-acsf-api-spec
```

Check that it updates the ACSF openAPI spec (we should be able to see "sane" openAPI contents, and there should be at least some minimal change from the last known spec)

1. Follow the [contribution guide](https://github.com/acquia/cli/blob/master/CONTRIBUTING.md#building-and-testing) to set up your development environment.
2. Clear the kernel cache to pick up new and changed commands: `./bin/acli ckc`
3. (add specific steps for this pr)

**Merge requirements**
